src/components/AppLayout/TopNav/TopNavLink.tsx

```diff
@@ -1,12 +1,12 @@
-import { ArrowLeftIcon, Button, sprinkles } from "@saleor/macaw-ui/next";
+import { ArrowLeftIcon, Button } from "@saleor/macaw-ui/next";
 import React from "react";
 import { Link } from "react-router-dom";
 
 export const TopNavLink: React.FC<{
   to: string;
   variant?: "secondary" | "tertiary";
 }> = ({ to, variant = "secondary" }) => (
-  <Link to={to} className={sprinkles({ marginRight: 2 })}>
```

Comment:
**issue**: This change breaks UI in multiple places of dashboard. e.g. product details, order details, customer details. Example:
<img width="364" alt="image" src="https://github.com/saleor/saleor-dashboard/assets/41952692/c8cb2818-fc60-4f4f-adc8-815224f8ff28">

Reply:
I've restored the original styling